src/server/controllers/recommendation.js

```diff
@@ -0,0 +1,87 @@
+const { prisma } = require('../utils/prisma');
+const { SERVER_SUCCESS, SERVER_ERROR } = require('../config.js');
+
+const createRecommendation = async(req, res) => {
+    const userId = Number(req.params.userId);
+    const { bookId, isStored } = req.body;
+
+    const recommendation = {
+        bookId: Number(bookId),
+        userId: userId,
+        isStored,
+        isMatch: false
+    }
+
+    const existingRecommendation = await prisma.recommendation.findFirst({
```

Comment:
I think this should be inside the `try / catch` block as well - if something goes wrong with prisma, you'll want to handle that by sending your error response back to the client

Reply:
It was inside a try / catch before. And the server crashed because of it.